### ISSUE TITLE
[FEATURE RQ]  Profiles for remapping

### ISSUE THREAD

**Issue by MightyCrusier** [Aurora-RGB/Aurora#235]
When using remapping for RGB.NET devices it would be nice if we could have the ability to bind a certain remap setup to a profile.

Example:
When wanted in GTA V the F keys light up as police lights - In this case I want my lightstrip and PC lights to remap to F keys

In Cyberpunk 2077 number keys represent a health bar - In this case I want my lightstrip and PC lights to remap to number keys

It would be easier if there was a remap layer or remap profiles.

**Comment by Aytackydln** [comment]
You can already remap integration layers

**Comment by MightyCrusier** [comment]
Yes, but when you want to remap to a different layout, you have to do it one led at a time, which is tedious. Wouldn't it be better if you could make profiles for remapping, that switched automatically on per-game basis?
